frontend/src/components/ShareBox.vue

```diff
@@ -0,0 +1,153 @@
+export default {
+  name: 'ShareBox',
+  data() {
+    return {
+      textCopy: 'Copier l\'URL',
+      textCopied: 'URL copiée',
+    };
+  },
+  props: {
+    quiz: String, // Add the component to question later
+    quizName: String,
+    score: String,
+  },
+  computed: {
+    quizShareUrl() {
+      // return window.location.origin + this.$route.path;
+      // In order to test in local we need to have a valid url (not localhost)
+      return `https://quizanthropocene.fr${this.$route.path}`;
+    },
+    twitterDataText() {
+      // On peut mettre un message plus engageant avec le score réaliser, le titre du quiz, ...
+      return `${this.quizName}`;
+    },
+  },
+  mounted() {
+    // Add Twitter script
+    const twitterScript = document.createElement('script');
+    twitterScript.setAttribute('src', 'https://platform.twitter.com/widgets.js');
+    twitterScript.setAttribute('charset', 'utf-8');
+    document.head.appendChild(twitterScript);
+
+    // Add Facebook Script
+    const fbScript = document.createElement('script');
+    fbScript.text = `(function(d, s, id) {
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = "https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v3.0";
+        fjs.parentNode.insertBefore(js, fjs);
+      }(document, 'script', 'facebook-jssdk'));`;
+    document.body.appendChild(fbScript);
+  },
+  methods: {
+    copyQuizShareUrlToClipboard() {
+      const copyText = document.getElementById('quiz-share-url');
+      copyText.select();
+      const oldValue = this.textCopy;
+      this.textCopy = this.textCopied;
+      document.execCommand('copy');
+      setTimeout(() => {
+        this.textCopy = oldValue;
+      }, 2000);
+    },
+    shareLinkedIn() {
+      const url = `https://www.linkedin.com/shareArticle?mini=true&url=${this.quizShareUrl}`;
+      window.open(url, '_blank');
+    },
+  },
+
+};
+</script>
+<style scoped>
+
+#quiz-share-url {
+  /* display: none; */
+  background: transparent;
+  border: none;
+  position: fixed;
+  left: -1000px;
+}
+input:focus{
+  outline: none;
+}
+
+.share-buttons {
+  margin-left: 20px;
+  margin-top: 10px;
+}
+.share-button {
+  margin-left: 10px;
+}
+.share-button-url {
+  vertical-align: middle;
+}
+.linkedin-button {
+  /* I kept the value from the linkedIn real share button */
+  background-color: #0073b1;
+  height: 28px;
+  padding: 0 10px 0 8px;
+  border: none;
+  border-radius: 0.3rem;
+  color: #fff;
+  display: inline-block;
+  font-family: -apple-system,system-ui,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Fira Sans,Ubuntu,Oxygen,Oxygen Sans,Cantarell,Droid Sans,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Lucida Grande,Helvetica,Arial,sans-serif!important;
+  font-weight: 600;
+  overflow: hidden;
+  outline-width: 2px;
+  position: relative;
+  text-align: center;
+  text-decoration: none;
+  vertical-align: middle;
```

Comment:
```suggestion
  vertical-align: sub;
```
Un chouia mieux je trouve